languages/common-lisp/exercises/concept/lillys-lasagna-leftovers/.docs/instructions.md

```diff
@@ -0,0 +1,109 @@
+Lilly the Lisp Alien has definitely made a lot of Lasagna with your
+help. They want to thank you by giving you some leftovers. But first
+they have some feedback about one of the functions you helped them
+with.
+
+In this exercise you are going to write some more functions to help
+Lilly with their Lasagna cooking as well as helping them split the
+leftovers with you.
+
+You have six tasks, three about improving the existing Lasagna cooking
+functions and three about splitting the leftovers.
+
+## 1. Make `preparation-time-in-minutes` easier to use
+
+Lilly says they found using your `preparation-time-in-minutes` a
+little awkward to use since they had to count all the layers before
+using the functions. Lilly wonders if you could make a version of the
+function that takes the layers (any number of them) as arguments and
+then calculate the preparation time based upon how many layers there
+were? As a reminder, it takes 19 minutes to prepare a single layer.
+
+```lisp
+(preparation-time 'sauce 'cheese 'right-handed-macaroni 'cheese 'sauce
+                  'left-handed-macaroni 'sauce 'sauce 'cheese 'cheese)
+                  ;; => 190 (because there are 10 layers.
+```
+
+## 2. Allow changing the expected oven time
+
+While Lilly's parental-units always cooked their lasagna for 337
+minutes, Lilly does admit that other Lisp Aliens like to cook their
+lasagna for longer or shorter amounts of time.
+
+So it would be good if `remaining-minutes-in-oven` would not use
+`expected-minutes-in-oven` but instead could take an _optional_
+parameter to define if the time should be longer, very long, shorter
+or very short. Longer or shorter means add or subtract 100 from the
+default time of 337. Very long or very short means add or
+subtract 200. If no parameter is provided, or the argument is 'normal'
+then the default value of 337 should be returned.
+
+```lisp
```

Comment:
We probably don't need to show every case, but maybe we should change the last example to `:very-long` (or however it should be formatted). That would dispel the ambiguity between `:verylong`, `:very-long`, `:|very long|` etc without needing to check the tests.